commodore/dependency_mgmt.py

```diff
@@ -84,27 +84,43 @@ def _discover_components(cfg, inventory_path):
     return sorted(components), component_aliases
 
 
-def _read_component_urls(cfg: Config, component_names) -> Dict[str, str]:
+def _read_components(
+    cfg: Config, component_names
+) -> Tuple[Dict[str, str], Dict[str, Optional[str]]]:
     component_urls = {}
+    component_versions = {}
 
-    if cfg.debug:
-        click.echo(f"   > Read commodore config file {cfg.config_file}")
-    try:
-        commodore_config = yaml_load(cfg.config_file)
-    except Exception as e:
-        raise click.ClickException(
-            f"Could not read Commodore configuration: {e}"
-        ) from e
+    reset_reclass_cache()
```

Comment:
We should probably provide a helper function to get the inventory and make sure reset cache is called.